pandas/tests/tslibs/test_array_to_datetime.py

```diff
@@ -170,7 +170,7 @@ def test_to_datetime_barely_out_of_bounds():
     # Close enough to bounds that dropping nanos
     # would result in an in-bounds datetime.
     arr = np.array(["2262-04-11 23:47:16.854775808"], dtype=object)
-    msg = "Out of bounds nanosecond timestamp: 2262-04-11 23:47:16"
+    msg = "Out of bounds .* present at position 0"
```

Comment:
Could you format one matching pattern with the exact string to check that it works correctly?